qiskit/transpiler/passes/optimization/convert_discrete_rotations.py

```diff
@@ -0,0 +1,46 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2025
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Convert rotation gates into {Clifford,T,Tdg} when their angles are integer multiples of 2*pi/8"""
+
+from qiskit.transpiler.basepasses import TransformationPass
+from qiskit.dagcircuit import DAGCircuit
+from qiskit._accelerate.discretize_rotations import discretize_rotations
+
+
+class DiscretizeRotations(TransformationPass):
+    """Convert rotation gates into {Clifford,T,Tdg} when their angles are integer multiples of 2*pi/8."""
```

Comment:
Why not write `pi/4` instead? 😄 

Should we also say "single-qubit" rotation gates, or even more explicitly RX, RY and RZ gates?

Reply:
I don't feel strongly about this, but with Qiskit convention, $T$ is equivalent to $RZ(\pi/4)$ (up to a global phase), so I do think that $(\pi/4)$-rotation is clearer than $(2\pi/8)$-rotation. In any case, since there are at least two different conventions in the literature, it might be good to slightly expand the documentation (here and in other relevant places) to eplicitly say that rotations by multiples of $(\pi/2)$ (or $4\pi/8$, if you prefer) are Clifford, while rotations by (odd) multiples of $\pi/4$ are Clifford+T.